nemo-physical/src/dictionary/meta_dv_dict.rs

```diff
@@ -305,20 +305,23 @@ impl MetaDvDictionary {
                     return;
                 }
                 dict = Box::new(LangStringDvDictionary::new());
+                //dict = Box::new(LangStringSplittingDvDictionary::new());
```

Comment:
Well, it is the place where one can drop in the old dict if desired for testing. Since the code is still there (also commented out in the module file), I thought it makes sense to handle the actual use similarly.

Reply:
Swapping the dictionaries also requires changing the imports, so it's not that this would be a single-line change, but fair enough if you absolutely want to keep it.